src/functions.jl

```diff
@@ -595,14 +635,7 @@ for T in (:Taylor1, :TaylorN)
             x = a[1]
             zero!(s, k)
             zero!(c, k)
-            if $T == Taylor1
-                @inbounds s[k] = x * c[k-1]
-                @inbounds c[k] = x * s[k-1]
-            else
-                @inbounds mul!(s[k], x, c[k-1])
-                @inbounds mul!(c[k], x, s[k-1])
-            end
-            @inbounds for i = 2:k
+            @inbounds for i = 1:k
```

Comment:
Same comment about order of `if` and `for`

Reply:
Done 👍